### DISCUSSION TITLE
Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson support for net472

### DISCUSSION THREAD

**Discussion by mkovacina** [dotnet/aspnetcore#26607] [Q&A] [answered]
Is it a supported use case that `Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson 3.1.x ` can be used with a full framework application?

**background**
I have a `net472` project that needs to communicate to a `netcoreapp3.1` process via SignalR.  The `netcoreapp3.1` process is using `Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson` and thus my project must also use the library as well.  

As there is no 2.1.x version of `Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson`, my `net472` project now pulls in 3.1.x versions of `Microsoft.Extentions.*` dependent projects.  As these all are `netstandard2.0` my project compiles and functions, but I am concerned about support for this use case.

Any clarification or guidance would be appreciated.

**Comment by BrennanConroy** [comment, marked as answer]
>Is it a supported use case that Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson 3.1.x can be used with a full framework application?

Yes, otherwise we wouldn't target netstandard and instead target netcoreappX.Y. We have also added NET461 to the target frameworks in 5.0.

**Comment by mkovacina** [comment]
Thank you Brennan.  I appreciate the response and the validation of the use case.
